### ISSUE TITLE
Ninja and CUDA v11.3

### ISSUE THREAD

**Issue by lfbarba** [Ruyi-Zha/naf_cbct#26]
I have been getting many errors when trying to run the code. Even with CUDA 11.3 installed, ninja seems to have issues with its installation and the code fails. To avoid these issues that are also documented for CUDA 11.3 in other applications, I was wondering if you have a container that can run this code, maybe one in docker hub ?
thanks

**Comment by Ruyi-Zha** [comment]
I apologize for the inconvenience; we don't have a container for that environment. We test our code on Ubuntu 20.04 and haven't encountered installation errors on our end.

Since we primarily use common Python packages, there shouldn't be many issues. Could you please share the error details? I'll do my best to help troubleshoot the problem.

**Comment by yarkinslav** [comment]
This issue is connected with hashencoder.cu described here https://github.com/ashawkey/torch-ngp/issues/4. This issue was solved with enabling fp16 in NerfRenderer. This repository do not support older architecture GPUs fp16.

**Comment by Erhushenshou** [comment]
Did you met problems with disutils.core file in setup.py?

**Comment by yarkinslav** [comment]
@Erhushenshou No, but I was not able to solve it with recommended changes in cuda code hashencoder.cu.

**Comment by robot131** [comment]
There was a issue connected with hashencoder.cu described here [https://github.com/NVlabs/instant-ngp/issues/119](url). And my problem that is related to the error:
`error: parameter packs not expanded with ‘...’`
was solved by the operation [https://github.com/NVlabs/instant-ngp/issues/119#issuecomment-1034701258](url)

Ps: if the permission is denied when you run the 'ln' lines, you can add 'sudo' command behind the 'ln', and commonly it will work
